service/drempelbox.service

```diff
@@ -1,17 +1,17 @@
 [Unit]
 Description=drempelbox
-Requires=sound.target user@1000.service sys-devices-platform-soc-3f980000.usb-usb1-1\x2d1-1\x2d1:1.0-sound-card0-controlC0.device
-After=sound.target user@1000.service sys-devices-platform-soc-3f980000.usb-usb1-1\x2d1-1\x2d1:1.0-sound-card0-controlC0.device
+Requires=sound.target user@107.service sys-devices-platform-soc-3f980000.usb-usb1-1\x2d1-1\x2d1:1.0-sound-card0-controlC0.device
```

Comment:
No that is the part which is still a bit unsure: how to get the UID of user `drempelbox` reliably in case it ends up not being 107 on another system.
The other manual steps are the ones in the readme. Setting up a polkit policy, adding the user and group membership etc.

Reply:
How averse are you to use the `system` library? You could then just query for `id -u drempelbox`. But I guess you know that.

Reply:
i don't think we can do this easily within the unit file though. I can see two way:
If we can use env vars in the unit file, we can have a script that exports the UID into an env var on boot, but this has to happen _before_ systemd runs. Not sure how feasible this is.
Or hope the UID doesn't change once setup (i guess that's kind of reasonable for our use case) and then have a small script that creates the unit file from a template whenever we "install" the service. It can then insert the correct UID.

Reply:
So i came up with a third way: i chose not to have this problem.
I just define a fixed UID (in a safe range, afaict) and use that :shrug: